internal/turso/turso.go

```diff
@@ -79,6 +82,13 @@ func (t *Client) newRequest(method, urlPath string, body io.Reader) (*http.Reque
 
 func (t *Client) do(method, path string, body io.Reader) (*http.Response, error) {
 	req, err := t.newRequest(method, path, body)
+	if flags.Debug() {
+		dump, err := httputil.DumpRequestOut(req, true)
+		fmt.Printf("%s", string(dump))
+		if err != nil {
+			return nil, err
```

Comment:
Let's not break the CLI because the dump functionality is not working properly! 😬
Just print the error and move on.